node/pkg/common/keys/keys.go

```diff
@@ -0,0 +1,29 @@
+package keys
+
+import (
+	"strconv"
+)
+
+func LatestFeedDataKey(feedID int32) string {
+	return "latestFeedData:" + strconv.Itoa(int(feedID))
```

Comment:
if feedID is already passed as an int32, i wonder why are casting it to int here?

Reply:
In Go, the strconv.Itoa function is designed to convert an int type to a string representation. It does not have built-in support for converting int32 or other integer types directly.

😞